### PR TITLE
Allow crawling in robots.txt

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

**Review by ndushay — COMMENTED** [review]
will this keep "private" purls private due to sitemap?  If not, we can't do this.

**Comment by lwrubel** [comment]
Blocking indexing on everything else is handled by: https://github.com/sul-dlss/purl/blob/1cdb714d869f74169e07adf044ace4425918a2db/app/views/purl/show.html.erb#L17

File: public/robots.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # See http://www.robotstxt.org/wc/norobots.html for documentation on how to use the robots.txt file
 #
 # To ban all spiders from the entire site uncomment the next two lines:
-User-Agent: *
-Disallow: /
```
Comment (lwrubel) [inline public/robots.txt:5]: I think we want to Allow crawling, rather than Disallow?

Comment (justinlittman) [reply public/robots.txt:5]: This is commenting out the Disallow, which should translate into an implicit Allow.

Comment (lwrubel) [reply public/robots.txt:5]: Got it.

**Review by lwrubel — APPROVED** [review]

**Pull request merged**